### ISSUE TITLE
[Arm64] Strike change breaks tip

### ISSUE THREAD

**Issue by sdmaclea** [dotnet/coreclr#13405]
/home/vmjenkins/workspace/Dotnet/build_and_test/src/ToolBox/SOS/Strike/strike.cpp:14676:59: error: no member named 'Rip' in '_CONTEXT'
    HRESULT hr = g_ExtSymbols->GetModuleByOffset(context->Rip, 0, NULL, &baseAddress);
                                                 ~~~~~~~  ^
/home/vmjenkins/workspace/Dotnet/build_and_test/src/ToolBox/SOS/Strike/strike.cpp:14681:54: error: no member named 'Rip' in '_CONTEXT'
            if (frame->InstructionOffset == context->Rip)
                                            ~~~~~~~  ^
/home/vmjenkins/workspace/Dotnet/build_and_test/src/ToolBox/SOS/Strike/strike.cpp:14725:21: error: no member named 'Rip' in '_CONTEXT'
        if (context.Rip == 0)
            ~~~~~~~ ^
/home/vmjenkins/workspace/Dotnet/build_and_test/src/ToolBox/SOS/Strike/strike.cpp:14729:49: error: no member named 'Rip' in '_CONTEXT'
        nativeFrame.InstructionOffset = context.Rip;
                                        ~~~~~~~ ^
/home/vmjenkins/workspace/Dotnet/build_and_test/src/ToolBox/SOS/Strike/strike.cpp:14730:44: error: no member named 'Rip' in '_CONTEXT'
        nativeFrame.ReturnOffset = context.Rip;
                                   ~~~~~~~ ^
/home/vmjenkins/workspace/Dotnet/build_and_test/src/ToolBox/SOS/Strike/strike.cpp:14731:43: error: no member named 'Rbp' in '_CONTEXT'
        nativeFrame.FrameOffset = context.Rbp;
                                  ~~~~~~~ ^
/home/vmjenkins/workspace/Dotnet/build_and_test/src/ToolBox/SOS/Strike/strike.cpp:14732:43: error: no member named 'Rsp' in '_CONTEXT'
        nativeFrame.StackOffset = context.Rsp;
                                  ~~~~~~~ ^

**Comment by sdmaclea** [comment]
@jkotas @dotnet/arm64-contrib

**Comment by jkotas** [comment]
Fixed by #13402 ?

**Comment by sdmaclea** [comment]
Looks like it.
